docs/source/conf.py

```diff
@@ -17,9 +17,10 @@
 # add these directories to sys.path here. If the directory is relative to the
 # documentation root, use os.path.abspath to make it absolute, like shown here.
 #
-# import os
-# import sys
-# sys.path.insert(0, os.path.abspath('.'))
+import os
```

Comment:
Why do we uncomment now?

Reply:
To have sphinx to reference the source code first, rather than going directly to the build directory; this allows for changes to be previewed using sphinx-autobuild command before they are committed.